packages/opentelemetry-exporter-collector-grpc/src/util.ts

```diff
@@ -32,6 +32,9 @@ export function onInit<ExportItem, ServiceRequest>(
   config: CollectorExporterConfigNode
 ): void {
   collector.grpcQueue = [];
+  if (collector.url.includes('/')) {
```

Comment:
Wouldn't this match the first `/` in the url? E.g. the one from `https://`?

Reply:
It might be better to parse the URL and just use the proto and host sections

Reply:
removeProtocol should handle this, I'll move it below removeProtocol.

Reply:
tests are still failing and I can't figure out why.

Reply:
I commented on the actual test. Seems like an issue with the test code not the implementation.